packages/node_modules/@webex/internal-plugin-calendar/test/unit/spec/calendar.js

```diff
@@ -0,0 +1,90 @@
+/*!
+ * Copyright (c) 2015-2019 Cisco Systems, Inc. See LICENSE file.
+ */
+
+import {assert} from '@webex/test-helper-chai';
+import Calendar from '@webex/internal-plugin-calendar';
+import MockWebex from '@webex/test-helper-mock-webex';
+import sinon from 'sinon';
+
+describe('internal-plugin-meetings', () => {
+  beforeEach(() => {
+    // LoggerConfig.set({verboseEvents: true, enable: false});
+    // LoggerProxy.set(logger);
+    // TriggerProxy.trigger = sinon.stub().returns(true);
+    // Metrics.initPayload = sinon.stub();
+    // Metrics.postEvent = sinon.stub();
+    // MediaUtil.createPeerConnection = sinon.stub().returns({});
+  });
+  describe('meeting index', () => {
+    const webex = new MockWebex({
+      children: {
+        calendar: Calendar
+      }
+    });
+
+    beforeEach(() => {
+      webex.canAuthorize = true;
+      webex.internal.device = {
+        register: sinon.stub().returns(Promise.resolve()),
+        unregister: sinon.stub().returns(Promise.resolve())
+      };
+      webex.internal.mercury = {
+        connect: sinon.stub().returns(Promise.resolve()),
+        disconnect: sinon.stub().returns(Promise.resolve()),
+        on: sinon.spy(),
+        off: sinon.spy()
+      };
+
+      webex.request = sinon.stub().returns(Promise.resolve({body: {items: [{id: 'calendar1'}, {id: 'calendar2'}]}}));
+    });
+    describe('Public Api Contract', () => {
+      describe('#register', () => {
+        it('should have #register', async () => {
+          await webex.internal.calendar.register();
+          assert.calledOnce(webex.internal.device.register);
+          assert.callCount(webex.internal.mercury.on, 5);
+          assert.equal(webex.internal.calendar.registered, true);
+        });
+      });
+
+      describe('#unregister', () => {
+        it('should have #unregister', async () => {
+          await webex.internal.calendar.unregister();
+          assert.callCount(webex.internal.mercury.off, 5);
+          assert.calledOnce(webex.internal.mercury.disconnect);
+          assert.calledOnce(webex.internal.device.unregister);
+        });
+      });
+
+      describe('#syncCalendar', () => {
+        it('should sync from calendar service', async () => {
+          await webex.internal.calendar.syncCalendar({fromDate: 'xx-xx', toDate: 'xx-nn'});
+          assert.calledWith(webex.request, {
+            method: 'GET',
+            service: 'calendar',
+            resource: 'calendarEvents',
+            qs: {fromDate: 'xx-xx', toDate: 'xx-nn'}
+          });
+          assert.equal(webex.internal.calendar.getAll().length, 2);
```

Comment:
What is this assertion for?

Reply:
checks the length of the calendar item returned